deno_webgpu/src/01_webgpu.js

```diff
@@ -154,15 +154,35 @@
     }
   }
 
-  class GPUOutOfMemoryError extends Error {
-    name = "GPUOutOfMemoryError";
+  class GPUError extends Error {
```

Comment:
unsure about correctness

Reply:
I don't have an answer, but I also don't think this would be a blocker for the PR